test/state/system_contracts.cpp

```diff
@@ -0,0 +1,74 @@
+// evmone: Fast Ethereum Virtual Machine implementation
+// Copyright 2023 The evmone Authors.
```

Comment:
```suggestion
// Copyright 2024 The evmone Authors.
```

Reply:
2023 is the year then this code was introduced.